mypy_requirements.txt

```diff
@@ -0,0 +1,2 @@
+mypy==0.902
```

Comment:
These are python3 requirements. So till python3 migration, @vojtechjelinek @sajalasati and I decided to keep it this way.

Reply:
> These are python3 requirements.

We are using mypy for the python2.7 type check, right? does it need python3 libs? (not sure!)

Also, in that case, we can call it py3_requirements.txt? (It can be used for other features which need py3 requirements and in future, we can replace it with requirements.txt) wdyt?

Reply:
Mypy can type check py2 code but it needs python3 to run. This was one of the main blockers of this PR. So had to make a lot of changes. And the name can remain same as there is a TODO to add these to the original requirements.txt file and this will remain same.

Reply:
I see, I'm fine to have it if we have TODO comment and github-issue for the same!

Reply:
https://github.com/oppia/oppia/issues/13131 Here is the issue. The comment is already added to mypy_requirements.txt file. PTAL.